strings/swap_case.py

```diff
@@ -0,0 +1,48 @@
+'''
+This algorithm helps you to swap cases.
+
+User will give input and then program will perform swap cases.
+
+In other words, convert all lowercase letters to uppercase letters and vice versa.
+For example:
+1)>>>Please input sentence: Algorithm.Python@89
+  aLGORITHM.pYTHON@89
+2)>>>Please input sentence: github.com/mayur200
+  GITHUB.COM/MAYUR200
+
+Constraints: Length of string should between 1 and 1000
+
+
+'''
+import re
+'''
+This re.compile() function saves the pattern from 'a' to 'z' and 'A' to 'Z' into 'regexp' variable
+'''
+regexp = re.compile('[^a-zA-Z]+')
```

Comment:
This is never used.

Reply:
I have used _regexp_   `if regexp.search(char):
            newstring += char` here